### DISCUSSION TITLE
Tailwind breaks when I re-run a next/react server

### DISCUSSION THREAD

**Discussion by AvivShvitzky** [tailwindlabs/tailwindcss#10687] [Help]
Tech: Tailwind latest version, next.js 12

My tailwind breaks (aka some classes suddenly have no effect, and apply no CSS properties) when I shut down my local server, and run it up again.
When I have my localhost opened on the browser,
and I re-run the server, the browser performs a re-render, and then the page is rendered broken.
I can fix it temporarily but making a change in the broken component, but the problem reappears when I re-run the server. It also is not fixed on pc restart or anything else

Is this a known issue?
[StackOverflow issue](https://stackoverflow.com/questions/67837075/tailwind-classes-not-working-after-page-refresh-in-production)

Have any thoughts?

**Comment by wongjn** [comment]
Consider proving a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by AvivShvitzky** [reply]
> Consider proving a remotely-hosted git repository that reproduces the unexpected behavior.

seems like I'm not the first one with such an issue?
https://stackoverflow.com/questions/67837075/tailwind-classes-not-working-after-page-refresh-in-production

**Reply by wongjn** [reply]
And probably won't be the last! Please consider proving a remotely-hosted git repository that reproduces the unexpected behavior and we can take a closer look at what might be the issue.
